src/term_image/renderable/_renderable.py

```diff
@@ -1017,7 +1017,7 @@ def _render_(self, render_data: RenderData, render_args: RenderArgs) -> Frame:
 
               * contain as many lines as :py:attr:`render_size.height
                 <term_image.geometry.Size.height>` i.e exactly
-                ``render_size.height - 1`` occurences of ``\\n`` (the newline sequence).
+                ``render_size.height - 1`` occurrences of ``\\n`` (the newline sequence).
```

Comment:
This makes the line length 89 (> 88) and causes the lint check to fail. The line will require rewrapping.